config/routes.rb

```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
-  devise_for :users, controllers: { :omniauth_callbacks => "omniauth_callbacks" }
   root to: 'welcome#index'
-  get '/auth/:provider/callback' => 'sessions#create'
-  get '/logout' => 'sessions#destroy', as: :logout
+
+  devise_for :users, :controllers => {
```

Comment:
__[rubocop]__
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)